synapse/lib/stormtypes.py

```diff
@@ -6455,6 +6471,10 @@ def _ts_to_reqdict(ts):
         if iden:
             cdef['iden'] = iden
 
+        view = kwargs.get('view')
+        if view:
```

Comment:
Lets make this automatically select the current view if unspecified ( and always set it in the cdef )

Reply:
:+1: